atomicapp/applogging.py

```diff
@@ -27,15 +27,20 @@
 class customOutputFormatter(logging.Formatter):
     """
     A class that adds 'longerfilename' support to the logging formatter
-    This 'longerfilename' will be filename + parent dir.
+    This 'longerfilename' will be path/to/file.py after the root atomicapp
+    folder.
     """
 
+    # setting the root directory path of code base
```

Comment:
Can you add some notes here about why this works.. i.e this works right now because this file exists at `atomicapp/applogging.py` which is directly under the root of atomicapp.